### ISSUE TITLE
error: SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode

### ISSUE THREAD

**Issue by yanivefraim** [wallabyjs/public#714]
I am using `node` environment, for loading my node tests (the application is an isomorphic react app, but I am currently trying to a separate `wallaby.js` files between client and server. 

I am getting the error for my test files (having `const` & `let` inside + using `"strict mode"`). When looking at my transpiled code (inside projectCacheDir) I see that the code is transpiled correctly (I have var instead of const).

### Wallaby.js configuration file
```javascript
module.exports = function (wallaby) {
  process.env.NODE_PATH += `:${require('path').join(wallaby.localProjectDir, 'node_modules')}`;

  return {
    files: [
      {pattern: 'node_modules/phantomjs-polyfill/bind-polyfill.js', instrument: false},
      {pattern: 'node_modules/es6-promise/dist/es6-promise.js', instrument: false},
      {pattern: 'node_modules/react/dist/react-with-addons.js', instrument: false},

      'src/**',
      {pattern: 'index.js', instrument: false},
      {pattern: 'test/environment.js', instrument: false},
      {pattern: 'test/routes/**/*.driver.js'},
      {pattern: 'package.json', instrument: false},
      {pattern: 'dist/**', instrument: false}
    ],

    tests: [
      {pattern: 'test/routes/**/*.spec.js', load: true}
    ],

    compilers: {
      '**/*.js*': wallaby.compilers.babel()
    },

    testFramework: 'jasmine',

    preprocessors: {
      'src/**/*.scss': file => require('node-sass').renderSync(
        { data: file.changeExt('css').content || ' ',
          includePaths: ['src/']
        }).css.toString()
    },

    env: {
      type: 'node',
      params: {
        env: `LOCAL_PATH=${process.cwd()}`
      }
    },
    workers: { recycle: true }
  };
};
```

### Code editor or IDE name and version

Atom v1.x

### OS name and version

OSX

**Comment by ArtemGovorov** [comment]
Could you please provide a sample repo or attach a small project where the issue is reproducible?

**Comment by yanivefraim** [comment]
Sorry for late response. It is hard for me to get a simple example for my case. I tried several node seeds (no react) - but couldn't reproduce my issue. 

I managed to solve this exact issue (required a file with no `'use strict';` inside. I now get:
` SyntaxError: Unexpected reserved word` error from that file. I guess this is not something related to Wallaby specifically, so I'll have to investigate it some more..
